crates/starknet-devnet-server/src/rpc_core/error.rs

```diff
@@ -94,6 +94,8 @@ pub enum ErrorCode {
     TransactionRejected,
     /// Custom geth error code, <https://github.com/vapory-legacy/wiki/blob/master/JSON-RPC-Error-Codes-Improvement-Proposal.md>
     ExecutionError,
+    /// method is forbidden for execution
+    MethodForbidden,
```

Comment:
Perhaps, from the users' perspective, to make the terminology clearer and more uniform, since we already mention the terms `restrictive` and `restricted`, we could replace `MethodForbidden` with `MethodRestricted` (similar for the message `"Method forbidden"`).